tests/testthat/output/trunc_mat/iris-3-5.txt

```diff
@@ -3,20 +3,21 @@
 #   150
 #   x
 #   5
-  Sepal.Length
-         <dbl>
-1          5.1
-2          4.9
-3          4.7
+ 
```

Comment:
@hadley: Do we need to care about very small output widths (here: 5 characters)?